column_chunk.go

```diff
@@ -25,6 +25,13 @@ type ColumnChunk interface {
 	// If the column chunk does not have a page index, the methods return nil.
 	ColumnIndex() ColumnIndex
 	OffsetIndex() OffsetIndex
+	BloomFilter() BloomFilter
+
+	// Returns the number of volumnes in the column chunk.
```

Comment:
```suggestion
	// Returns the number of values in the column chunk.
```

Values? Columns? Volumes?